flink-rpc/flink-rpc-akka/src/main/resources/META-INF/NOTICE

```diff
@@ -6,17 +6,17 @@ The Apache Software Foundation (http://www.apache.org/).
 
 This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
 
-- com.hierynomus:asn-one:0.5.0
+- com.hierynomus:asn-one:0.6.0
 - com.typesafe:config:1.4.2
 - com.typesafe:ssl-config-core_2.12:0.6.1
-- io.netty:netty:3.10.6.Final
-- org.agrona:agrona:1.15.1
-- org.apache.pekko:pekko-actor_2.12:1.0.1
-- org.apache.pekko:pekko-remote_2.12:1.0.1
-- org.apache.pekko:pekko-pki_2.12:1.0.1
-- org.apache.pekko:pekko-protobuf-v3_2.12:1.0.1
-- org.apache.pekko:pekko-slf4j_2.12:1.0.1
-- org.apache.pekko:pekko-stream_2.12:1.0.1
+- io.netty:netty-all:4.1.100.Final
```

Comment:
This should be changed to `4.1.91.Final` according to the [root POM](https://github.com/apache/flink/blob/60cba350d7638592ea771dc7cf512798e6248886/pom.xml#L369C14-L369C26).

Reply:
Nice catch! Changed this now :)